src/js/views/patients/patient/patient.scss

```diff
@@ -10,16 +10,15 @@
   flex: 1;
   flex-direction: column;
   min-width: 744px;
-  padding-left: 24px;
+  padding: 0 24px;
   width: 100%;
 }
 
 .patient__sidebar {
   display: flex;
   flex: initial;
-  max-width: 400px;
   min-height: 0;
-  min-width: 256px;
+  width: 256px;
```

Comment:
Sets width of the sidebar to be `256px` no matter what.

Before the sidebar had a min width of `256px` and would extend to a max of `400px`. This would cause some UI alignment/visual issues when the patient's name is really long. As the long patient name would stretch the width of sidebar beyond `256px`.